scripts/update/01-run.sh

```diff
@@ -54,6 +44,12 @@ if [[ ! -z "${UMBREL_OS:-}" ]]; then
         echo "ERROR: No Umbrel installation found at SD root ${SD_CARD_UMBREL_ROOT}"
         echo "Skipping updating on SD Card..."
     fi
+    
+    # Update apt packages on update
+    DEBIAN_FRONTEND=noninteractive apt-get install unattended-updates -y
```

Comment:
Have you checked this definitely won't setup automatic background updates by default? There's no extra step we need to take to disable that?

Also would be great to guard this off with a check that only runs the install if the dep isn't already installed.

Reply:
From the documentation it seems like it only runs when configured to do so, I'll add a check